.assets/readme_chapters/Troubleshooting.md

```diff
@@ -0,0 +1,105 @@
+# Troubleshooting
```

Comment:
This collection of files will be the official public documentation. I think `doc` folder in the project root is the right place for it, but certainly not some hidden `.assets` folder. Documentation should be visible and easy to find.

Reply:
We can make this section a numbered list or use md headers `##` for titles, it is unclear when a problem/mitigation starts and ends.

Reply:
Addressed